packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -117,28 +181,28 @@ export const ContractType = (() => {
 
         if (!contract_type) return {};
 
-        let stored_barriers_data = {};
+        let stored_barriers_data: TStoredBarriersData = {};
```

Comment:
@kate-deriv since in this PR we still use `useTraderStores`, please add this types for `short_barriers` and `long_barriers` there, and simply use it here like this:

```suggestion
        let stored_barriers_data: TTradeStore['short_barriers' | 'long_barriers'] = {};
```

Reply:
Noted, thank you